app/import_functions.go

```diff
@@ -664,7 +664,7 @@ func (a *App) ImportUserTeams(user *model.User, data *[]UserTeamImportData) *mod
 		}
 
 		if member.ExplicitRoles != roles {
-			if _, err := a.UpdateTeamMemberRoles(team.Id, user.Id, roles); err != nil {
+			if _, err = a.UpdateTeamMemberRoles(team.Id, user.Id, roles); err != nil {
```

Comment:
See shadowing discussion re: conditional declarations, but this would traditionally be the counter-argument to my thoughts. In code that uses naked returns, re-declaring `err` here might obscure issues. But since we generally disallow naked returns in our code, I feel like this kind of change isn't meaningful and falls under the allow shadowing within conditional declarations.

Reply:
I not sure I get what do are saying. What do you mean by "conditional declarations"? Do you mean declaring `err` inside a `if`?